package.json

```diff
@@ -32,7 +32,6 @@
     "@nivo/bar": "0.59.0",
     "@typescript-eslint/eslint-plugin": "^2.29.0",
     "@typescript-eslint/parser": "^2.29.0",
-    "@zeit/next-sass": "^1.0.1",
```

Comment:
Les versions antérieures à Next nécessitaient d'utiliser ce module. Ce n'est plus le cas.
https://github.com/vercel/next.js/blob/master/errors/built-in-css-disabled.md